project/ISSUES.md

```diff
@@ -0,0 +1,171 @@
+which forces the reconciliation date.  Issues *may* be, but they don't
+necessarily have to be -- this is subjective and it depends on if the submitter
+(or any other contributor) wants to force the issue to be revisited at the next
+release.
+
+Effective use of milestones should prevent bugs from falling through the
+cracks, and will provide a mechanism for forcing the revisitation (and thus
+progress or even potential discard) of ideas for enhancements or features.
+
+
+Labeling Structure
+==================
+
+To allow for easy search, filtering, and general issue management every issue
+or PR (not tagged `Procedures` or `Planning`) is *required* to have three
+labels which indicate the type, status, and focus area of the issue.  Any issue
+without these three tags will be automatically have a `triage` label applied
+indicating that it needs human intervention to be correctly tagged.  These
+`triage` tagged issues will be regularly reviewed and tagged as appropriate.
+
+Type Labels
+-----------
+
+The 'class' label set is used for classifying the type of contribution or
+request/report to separate enhancements and new features from bugs, etc.
+
+* `class/bug` - something is broken, and it needs fixing
+* `class/documentation` - documentation is unclear or can be improved
+* `class/enhancement` - polish to an existing feature or interface
+* `class/feature` - something brand new
+
+Status Labels
+-------------
+
+The `status` of an issue should be tracked using the following stages:
+
+* `status/triage` - brand new issue/pr that doesn't offer a concrete plan or
+  solution
+* `status/planning` - issue reviewed and has a sufficiently detailed primary
+  message and/or commentary
+* `status/WIP` - this issue or PR is currently being worked on.  It should not
+  be merged (or closed) without pinging the owner/submitter.
+* `status/review` - issue is resolved or PR is complete and needs review
+
+Note that there are no `status/complete`, `status/wontfix`, `status/duplicate`,
+or other terminal status indicators.  This is intentional to keep the tail end
+of bookkeeping from getting onerous.  These sorts of terminal states and their
+justifications should be indicated in the closing comment by the issue closer.
+
+Focus Labels
+------------
+
+The 'focus' label is used for tagging issues and pull requests to a particular
+focus area.  This allows for easy searching within that particular domain, as
+well as more organized release notes.  Some examples, not-exhaustive, are here:
+
+* `focus/API`
+* `focus/cleanup`
+* `focus/jobs`
+* `focus/tests`
+* `focus/GIEs`
+* `focus/toolshed`
+* `focus/UI-UX`
+* `focus/workflows`
+
+This list will definitely grow over time.
+
+Other Useful Labels
+-------------------
+
+While the three labels sets indicating status, focus, and type are required
+there are several other labels that are be useful and/or have special purpose.
+
+* `Procedures` is a special tag that indicates that the issue is related to
```

Comment:
Yeah, I went with what we currently use as a label, but since this is an overhaul we should standardize now.